Production/python/Run2017A-PromptReco-v3/HTMHT_cff.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
2017A does not contain any certified data (and never will, afaik) - please remove